server/test/lib/gateway/gateway.test.js

```diff
@@ -1,1022 +0,0 @@
-const { expect } = require('chai');
```

Comment:
I see that you moved all the tests from the Gladys Gateway part into separate files

Did you just separate the tests without modifying them, or did you modify them ? 

If you modified them, it's going to be super hard to see the diff 😅

Reply:
I'll take time to ensure that I only split them.

Reply:
Sorry to ask this but this part is critical so I really want to review all the changes 😅